frontend/ui/quickstart.lua

```diff
@@ -218,6 +218,11 @@ The following methods are available for accessing your books and articles:
 * Favorites
 * History
 
+You can assign hotkeys for quick access to each of these dialogs.
+
+Note, our hotkeys plugin works independently for both our reader and file manager modules.
+You will need to set up the hotkeys for both modules separately.
```

Comment:
I'd make it something more like this. Primarily I'm not sold on "modules," and "file browser" is the name we settled on.
```suggestion
Note, the hotkeys plugin works independently for both the reader (book display) and file browser, and hotkeys will need to be set up for each individually.
```